app/controllers/state_file/questions/declined_terms_and_conditions_controller.rb

```diff
@@ -0,0 +1,12 @@
+module StateFile
+  module Questions
+    class DeclinedTermsAndConditionsController < QuestionsController
+
+      def self.show?(_intake)
+        false
+      end
```

Comment:
I think we should be storing their answer and whether they consented ([similar to GYR](https://github.com/codeforamerica/vita-min/blob/main/app/models/intake.rb#L380)) and that this should be shown if they have any answer other than "yes"